ci-operator/step-registry/baremetal/lab/pre/load-balancer/run/baremetal-lab-pre-load-balancer-run-commands.sh

```diff
@@ -65,13 +65,8 @@ podman run --name "haproxy-$CLUSTER_NAME" -d --restart=on-failure \
 
 echo "Setting the network interfaces in the HAProxy container"
 
-# For the given dhclient.conf, eth1 will also get default route, dns and other options usual for the main interfaces.
-# eth2 will only get local routes configuration
-if [ x"${DISCONNECTED}" != x"true" ]; then
-  devices=( eth1.br-ext eth2.br-int )
-else
-  devices=( eth1.br-int )
-fi
+devices=( eth1.br-ext eth2.br-int )
```

Comment:
later we get the `eth1` IP as API VIP, so we are now exposing clusters publicly and have the components use the external IP instead of the internal one.

Reply:
The changes have been implemented